cmd/gke-exec-auth-plugin/main.go

```diff
@@ -82,20 +89,40 @@ func main() {
 			klog.Exit(err)
 		}
 		token = tok.AccessToken
+		if tok.Expiry.IsZero() {
+			// Make Kubelet poke us every hour if the token didn't have an expiration
```

Comment:
the client is not just kubelet. ditto below

Reply:
fixed